Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -357,36 +357,53 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
         return indexPath
       }
 
-    // FB CAPI
+    // Facebook CAPI + Google Analytics
     let stripePaymentSheetDidAppear = self.stripePaymentSheetDidAppearProperty.signal
 
     _ = Signal.combineLatest(project, self.viewDidLoadProperty.signal)
       .takeWhen(stripePaymentSheetDidAppear)
       .observeValues { project, _ in
-        guard project.sendMetaCapiEvents else { return }
 
         AppEnvironment.current.appTrackingTransparency.updateAdvertisingIdentifier()
 
         guard let externalId = AppEnvironment.current.appTrackingTransparency.advertisingIdentifier
         else { return }
 
-        /** FIXME: Soon we will use `triggerThirdPartyEvents` mutation paired with an in-app flag for allowing an advertising identifier to be sent even if it isn't nil. That will affect `applicationTrackingEnabled` and `advertiserTrackingEnabled`. */
-
         _ = AppEnvironment
           .current
           .apiService
-          .triggerCapiEventInput(
+          .triggerThirdPartyEventInput(
             input: .init(
+              deviceId: externalId,
+              eventName: ThirdPartyEventInputName.AddNewPaymentMethod.rawValue,
               projectId: "\(project.id)",
-              eventName: FacebookCAPIEventName.AddNewPaymentMethod.rawValue,
-              externalId: externalId,
-              userEmail: AppEnvironment.current.currentUserEmail,
+              pledgeAmount: nil,
+              shipping: nil,
+              transactionId: nil,
+              userId: "\(AppEnvironment.current.currentUser?.id)",
               appData: .init(
                 advertiserTrackingEnabled: true,
                 applicationTrackingEnabled: true,
-                extinfo: ["i2"]
+                extinfo: [
+                  "i2",
+                  "",
+                  "",
+                  "",
+                  "\(ProcessInfo.processInfo.operatingSystemVersion)",
+                  "",
+                  "",
+                  "",
+                  "",
+                  "",
+                  "",
+                  "",
+                  "",
+                  "",
+                  "",
+                  ""
+                ]
```

Comment:
According to the CAPI documentation all values in `extInfo` are required and must be in the given indexed order.
I paired with Isa on this and this format is the only way we can get a successful 200 response for this mutation.

IMO we should only have to pass the used values (i2, OS version) to the backend and _they_ should be responsible for formatting the data correctly. 

Since the docs require this I chose to implement it this way on our end.

![Screenshot 2023-07-17 at 2 21 13 PM](https://github.com/kickstarter/ios-oss/assets/110618242/8b8d52ff-70fa-404a-ad70-4c6fcd712283)